package.json

```diff
@@ -33,7 +33,8 @@
     "prepare": "husky install",
     "prepublishOnly": "pinst --disable",
     "postpublish": "pinst --enable",
-    "test": "tap test"
+    "test": "tap test",
+    "taprun": "tap run"
```

Comment:
What does tap run do?

Reply:
`tap run` lets me specify which test file I want to run, instead of running all of them at once. I tried doing `tap test` and specifying a single file or module, but I couldn't get it to work. I can remove it after I am done.